packages/xchain-cosmos/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@xchainjs/xchain-cosmos",
-  "version": "0.17.0",
+  "version": "0.17.1",
```

Comment:
Let's bump to an alpha version `0.17.1-alpha1` (or by introducing breaking changes `0.18.0-alpha1`) first, so other contributors of projects can add more changes needed for ATOM until we publish an official release. I know someone who is currently on ASGDX on same things as well lol

Reply:
Fixed in #599